### DISCUSSION TITLE
brew cu starting problem

### DISCUSSION THREAD

**Discussion by kikieri** [Homebrew/discussions#3281] [Getting started]
Hello guys,
thanks for this awesome project!
I got since today the problem when I wanna start "brew cu" to update my casks.
Got the following Error:

Error: No such file or directory @ rb_sysopen - /usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/addressable-2.8.0/data/unicode.data

brew doctor is no problem,
tried also brew update-reset, mac-cleanup etc. 

Any idea whats thew problem?

Thanks in advance!

**Comment by gromgit** [comment]
> I got since today the problem when I wanna start "brew cu" to update my casks.

```
$ brew cu
Error: Unknown command: cu
```
The `cu` subcommand is not standard Homebrew. Please **show** us the code that implements that subcommand.

> brew doctor is no problem,

Please **show** us the actual output of `brew config` and `brew doctor`, instead of **telling** us what you think the output means.
